run-benchmarks.sh

```diff
@@ -115,6 +115,17 @@ function setup {
   cd benchmarks
 }
 
+# Record CPU and memory usage
```

Comment:
I'm going to add a command argument to only take CPU/MEM usage if it's necessary.